src/scripts/main.js

```diff
@@ -1 +1,130 @@
 'use strict';
+
```

Comment:
you have a huge js file. You can create small files

Reply:
Just create small files and include them in HTML.? Or should I use export/import for main.js?

Reply:
import on main 👌